### PR TITLE
doc: create stone deb installation guide

### PR DESCRIPTION
**Pull request by od-hunter**
#25

### CODE REVIEW

**Comment by od-hunter** [comment]
@dmirgaleev  

this is a draft because am having a minor challenge installing and using the stone package on my local machine.

# what I have done sofar
- I successfull downloaded the stone-prover-linux-x86_64.deb package which is compactable with my system architecture.

- I tried install using  ` sudo dpkg -i stone-prover-linux-x86_64.deb` and 
tried updating the dependencies using `sudo apt-get install -f`

- after the regular installation process trying to use the binary files:
`stone-prover --version` or `/usr/bin/cpu_air_verifier --version` or `/usr/bin/cpu_air_prover --version`   generates "command not found"

**Kindly assist**

**Review by dmirgaleev — CHANGES_REQUESTED** [review]

**Comment by od-hunter** [comment]
Hi @dmirgaleev all your corrections have been implement.

and thanks for the pointer it works fine now for performing the test now.

Kindly review

File: docs/pages/install/debian.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+```
+
+## Step 2: Installating the Package.
+
+**Install the package:** Run the following command to install Stone:
+```bash
+sudo dpkg -i /tmp/stone-prover-linux-x86_64.deb 
+```
+
+**Enter your password:** When prompted, enter your user password to proceed with the installation:
```
Comment (dmirgaleev) [inline docs/pages/install/debian.md:26]: I think it's unnecessary step, let's delete that

File: docs/pages/install/debian.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+``` 
+
+**Navigate to Test Files Directory:** Change to the directory where the test files are located. This is where you will find the input files needed for running the proof.
+```bash
+cd /tmp/stone-packaging/test_files 
+```
+
+**Run the Proof**: Execute the proof command using the cpu_air_prover. This command generates a proof file in the specificed `--out_file` option value(fibonacci_proof.json).
+```bash
+cpu_air_prover --out_file=fibonacci_proof.json --private_input_file=fibonacci_private_input.json --public_input_file=fibonacci_public_input.json --prover_config_file=cpu_air_prover_config.json --parameter_file=cpu_air_params.json
```
Comment (dmirgaleev) [inline docs/pages/install/debian.md:48]: Let's format that like it is here https://github.com/dipdup-io/stone-packaging/blob/master/README.md#creating-and-verifying-a-test-proof-using-binaries

File: docs/pages/install/debian.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+cd /tmp/stone-packaging/test_files 
+```
+
+**Run the Proof**: Execute the proof command using the cpu_air_prover. This command generates a proof file in the specificed `--out_file` option value(fibonacci_proof.json).
+```bash
+cpu_air_prover --out_file=fibonacci_proof.json --private_input_file=fibonacci_private_input.json --public_input_file=fibonacci_public_input.json --prover_config_file=cpu_air_prover_config.json --parameter_file=cpu_air_params.json 
+```
+
+**Verify the Proof:** After generating the proof, verify it using the cpu_air_verifier. This command checks the validity of the proof and outputs a success message if the verification passes.
+cpu_air_verifier --in_file=fibonacci_proof.json && echo "Successfully verified example proof."
```
Comment (dmirgaleev) [inline docs/pages/install/debian.md:52]: Let's format that like it is here [master/README.md#creating-and-verifying-a-test-proof-using-binaries](https://github.com/dipdup-io/stone-packaging/blob/master/README.md#creating-and-verifying-a-test-proof-using-binaries)

**Review by dmirgaleev — CHANGES_REQUESTED** [review]

**Comment by od-hunter** [comment]
Requested change has been implemented @dmirgaleev

File: docs/pages/install/debian.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+This guide provides instructions to install Stone from a `.deb` package on Ubuntu or Debian.
+
+## Prerequisites
+
+- A Debian-based system (recommended: Ubuntu 22.04 or Debian 11; minimum supported: Ubuntu 18.04 or Debian 9)
+- Super user Privilege (sudo) access to install packages
+
+## Step 1: Download the Stone `.deb` Package
+
+Download the latest .deb package from the Stone Packaging Releases page to the /tmp directory. Alternatively, you can use `wget` to download it directly.
```
Comment (dmirgaleev) [inline docs/pages/install/debian.md:12]: What does that mean? I believe there's only one way.

File: docs/pages/install/debian.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+## Step 2: Installating the Package.
+
+**Install the package:** Run the following command to install Stone:
+```bash
+sudo dpkg -i /tmp/stone-prover-linux-x86_64.deb 
+```
+
+
+## Step 3: Implement Run Test Proof
```
Comment (dmirgaleev) [inline docs/pages/install/debian.md:27]: As I said, that will be covered separately. Please, remove

**Review by dmirgaleev — CHANGES_REQUESTED** [review]

**Comment by od-hunter** [comment]
@dmirgaleev , please review.

**Pull request merged**